clowdapp.yaml

```diff
@@ -0,0 +1,167 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: drift-backend
+
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: drift-backend
+    labels:
+      app: drift-backend
+  spec:
+    envName: ${ENV_NAME}
+    deployments:
+    - name: service
+      minReplicas: ${{MIN_REPLICAS}}
+      webServices:
+        public:
+          enabled: true
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        livenessProbe:
+          httpGet:
+            path: /mgmt/v0/status
+            port: 8080
+            initialDelaySeconds: 10
+            periodSeconds: 10
+        readinessProbe:
+          httpGet:
+            path: /mgmt/v0/status
+            port: 8080
+            initialDelaySeconds: 10
+            periodSeconds: 10
+        resources:
+          limits:
+            cpu: ${CPU_LIMIT}
+            memory: {MEMORY_LIMIT}
+          requests:
+            cpu: ${CPU_REQUEST}
+            memory: ${MEMORY_REQUEST}
+        volumes:
+          - emptyDir: {}
+            name: drift-prometheus-data
+        volumeMounts:
+          - mountPath: /prometheus-data
+            name: drift-prometheus-data
+        env:
+          - name: APP_CONFIG
+            value: gunicorn.conf.py
+          - name: BASELINE_SVC_URL
+            value: "http://${SYSTEM_BASELINE_HOST}:8080"
+          - name: HSP_SVC_URL
+            value: "http://${HSP_HOST}:8080"
+          - name: RBAC_SVC_URL
+            value: "http://${RBAC_HOST}:8080"
+          - name: ENABLE_RBAC
+            value: "${DRIFT_ENABLE_RBAC}"
+          - name: ENABLE_SMART_MANGEMENT_ENTITLEMENT_CHECK
+            value: "${ENABLE_SMART_MANGEMENT_ENTITLEMENT_CHECK}"
+          - name: WEB_CONCURRENCY
+            value: "4"
+          - name: PATH_PREFIX
+            value: /api/
+          - name: APP_NAME
+            value: drift
+          - name: ENABLE_PIPENV
+            value: 'true'
+          - name: prometheus_multiproc_dir
+            value: /prometheus-data
+          - name: INVENTORY_SVC_URL
+            value: "http://${HOST_INVENTORY_HOST}:8080"
+          - name: CW_LOG_GROUP
+            value: "${CLOUD_WATCH_LOG_GROUP}"
+          - name: AWS_REGION
+            value: 'us-east-1'
+          - name: CW_AWS_ACCESS_KEY_ID
+            valueFrom:
+              secretKeyRef:
+                key: CW_AWS_ACCESS_KEY_ID
+                name: cloudwatch
+          - name: CW_AWS_SECRET_ACCESS_KEY
+            valueFrom:
+              secretKeyRef:
+                key: CW_AWS_SECRET_ACCESS_KEY
+                name: cloudwatch
```

Comment:
Cloudwatch config is also set by Clowder, so you can also drop this section . Clowder should provide the `access key`, the `region`, the `secret access key` and the `log group`.